.circleci/config.yml

```diff
@@ -144,7 +144,7 @@ jobs:
       - run:
           name: Run e2e navigation test
           command: |
-            python -m scripts.run_e2e_tests --prod_env --skip-install --suite="navigation"
+            python -m scripts.run_e2e_tests --circleci_env --skip-install --suite="navigation"
```

Comment:
Instead of having this flag, please have a flag that simply denotes that you are using a specific version of a config. Another thing to consider is whether TravisCI would benefit, that would be preferable so that we have one config instead of two.

Reply:
I didn't get that. Can you give example of such a flag? 

I don't know about TravisCI, but I think in local env it would make the things faster. What are your thoughts @vojtechjelinek?

Reply:
I'm not sure what it would do with the compile-time on Travis or locally, I think that there is a reason why parallelism is enabled by default, so without proper testing, I wouldn't extend it to other places right now.

Reply:
The frontend coverage flag is one example of such flag. We intended it to run only on CircleCI, but we didn’t label the flag that way. The reason is that we want to be able to use the flag in other environments if we need to.

Reply:
@kevinlee12 Done! I changed the flag to `--deparallelize-terser`.